File: repos_estudiantes_TARQUI_primera_entrega/LUNA_CRUZ_CAMILA_MILENKA/Ejercicio5/Estudiante.py
#5.Crea una clase Estudiante con nombre, nota_1,nota_2
class Estudiante:
    def __init__(self, nombre, nota_1, nota_2):
        self.nombre = nombre
        self.nota_1 = nota_1
        self.nota_2 = nota_2

    def calcular_promedio(self):
        return (self.nota_1 + self.nota_2) / 2

    def verificar_aprobo(self):
        if self.calcular_promedio() >= 6:
            print(f"EL ESTUDIANTE {self.nombre} APROBO")
        else:
            print(f"EL ESTUDIANTE {self.nombre} REPROBO")

    def mostrar(self):
        print(f"Nombre: {self.nombre} nota_1: {self.nota_1} nota_2: {self.nota_2}")

if __name__ == "__main__":
    estudiante1 = Estudiante("Carlos Gutierrez", 6, 6)
    estudiante2 = Estudiante("Carolina Mamani", 7, 8)
    estudiante3 = Estudiante("Pedro Calle", 2, 8)

    estudiante1.mostrar()
    estudiante2.mostrar()
    estudiante3.mostrar()

    print("-----NOTAS--------")
    print(f"{estudiante1.nombre} : Promedio: {estudiante1.calcular_promedio()}")
    estudiante1.verificar_aprobo()
    print(f"{estudiante2.nombre} : Promedio: {estudiante2.calcular_promedio()}")
    estudiante2.verificar_aprobo()
    print(f"{estudiante3.nombre} : Promedio: {estudiante3.calcular_promedio()}")
    estudiante3.verificar_aprobo() 

File: repos_estudiantes_TARQUI_primera_entrega/LUNA_CRUZ_CAMILA_MILENKA/Ejercicio5/Estudiante.java
package Ejercicio5;
//5.Crea una clase Estudiante con nombre, nota_1,nota_2
public class Estudiante {
    public String nombre;
    public int nota_1;
    public int nota_2;

    public Estudiante(String nombre,int nota_1,int nota_2){
        this.nombre=nombre;
        this.nota_1=nota_1;
        this.nota_2=nota_2;
    }
    public double calcularPromedio(){
        return(nota_1+nota_2)/2;
    }
    public void VerificarAprobo(){
        if(calcularPromedio()>=6){
            System.out.println("EL ESTUDIANTE " + nombre + "APROBO");
        }
        else{
            System.out.println("EL ESTUDIANTE " + nombre + " REPROBO");
        }
    }
    public void mostrar(){
        System.out.println("Nombre: " + nombre + " nota_1: " + nota_1 + " nota_2: " + nota_2);
    }
    public static void main(String[] args) {
        Estudiante estudiante1= new Estudiante("Carlos Gutierrez ", 6, 6);
        Estudiante estudiante2= new Estudiante("Carolina Mamani ", 7, 8);
        Estudiante estudiante3= new Estudiante("Pedro Calle ", 2, 8);
        estudiante1.mostrar();
        estudiante2.mostrar();
        estudiante3.mostrar();

        System.out.println("-----NOTAS--------");
        System.out.println(estudiante1.nombre + " : Promedio: " + estudiante1.calcularPromedio());
        estudiante1.VerificarAprobo();
        System.out.println(estudiante2.nombre + " : Promedio: " + estudiante2.calcularPromedio());
        estudiante2.VerificarAprobo();
        System.out.println(estudiante3.nombre + " : Promedio: " + estudiante3.calcularPromedio());
        estudiante3.VerificarAprobo();
    }
    
}

